enterprise/cli/templatecreate_test.go

```diff
@@ -1,143 +0,0 @@
-package cli_test
```

Comment:
why were these deleted?

Reply:
mistake from when I was removing the command